src/components/CreateRoom.tsx

```diff
@@ -47,14 +47,25 @@ export const CreateRoom: FC<Props> = ({ refetchIfNeeded, host }) => {
     });
   };
   const onChange = (event: ChangeEvent<HTMLInputElement>) => {
-    if (isRoomEnforceEncoding(event.target.value)) {
-      setEnforceEncodingInput(event.target.value);
-    }
+    setEnforceEncodingInput(isRoomEnforceEncoding(event.target.value) ? event.target.value : null);
   };
 
   return (
     <div className="card bg-base-100 shadow-xl indicator">
       <div className="card-body flex flex-row px-3 py-1 items-center">
+        <div className="form-control">
+          <label className="flex flex-row gap-2 label cursor-pointer items-center">
+            <span className="label-text">default</span>
```

Comment:
I would propose a different name for it. If a person is unfamiliar with the Jellyfish documentation, they might not know what it means.
 Maybe a name that suggests that both are acceptable or a checkbox instead.